components/PostComment.jsx

```diff
@@ -29,7 +28,7 @@ function PostComment({ articleId }) {
         setComment("");
         setCommentStatus("posted");
       } else {
-        throw new Error("User is not logged in.");
+        throw new Error("You are not authorized to post comments.");
```

Comment:
Currently when a user posts a comment, their comment isn't added in state, so is only available after a user has refreshed the page, which will need to be fixed 🔧 The reason I mention it in feedback for the delete ticket is that you should make sure when you do fix it that you're able to delete a comment that has just been created.